server/src/test/java/org/elasticsearch/gateway/ClusterStateUpdatersTests.java

```diff
@@ -268,7 +268,7 @@ public void testMixCurrentAndRecoveredState() {
         final ClusterState updatedState = mixCurrentStateAndRecoveredState(currentState, recoveredState);
 
         assertThat(updatedState.metaData().clusterUUID(), not(equalTo("_na_")));
```

Comment:
Should really use `UNKNOWN_CLUSTER_UUID` here and elsewhere in the tests too.

Reply:
see 3dc2c625e53